Doppler.Sap/Mappers/Billing/BillingForUsMapper.cs

```diff
@@ -156,13 +160,15 @@ public SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRequest b
             return sapSaleOrder;
         }
 
-        public SapIncomingPaymentModel MapSapIncomingPayment(int docEntry, string cardCode, decimal docTotal, string transferReference)
+        public SapIncomingPaymentModel MapSapIncomingPayment(int docEntry, string cardCode, decimal docTotal, string transferReference, DateTime? paymentDate)
         {
+            var date = paymentDate ?? _dateTimeProvider.GetDateByTimezoneId(_dateTimeProvider.UtcNow, _timezoneConfig.InvoicesTimeZone);
```

Comment:
what do you think add in this line the command .ToString("yyyy-MM-dd") ?

Reply:
Done